test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -9814,33 +9814,63 @@ def create_import_hook(self, name, attrs=None, body=None, overwrite=True,
         # In case of mom hooks, make sure that the hook related files
         # are successfully copied to the MoM
         try:
-            if 'exec' in attrs['event']:
-                hook_py = name + '.PY'
-                hook_hk = name + '.HK'
-                pyfile = os.path.join(self.pbs_conf['PBS_HOME'],
-                                      "server_priv", "hooks", hook_py)
-                hfile = os.path.join(self.pbs_conf['PBS_HOME'],
-                                     "server_priv", "hooks", hook_hk)
-                logmsg = hook_py + ";copy hook-related file request received"
-
-                cmd = os.path.join(self.client_conf['PBS_EXEC'], 'bin',
-                                   'pbsnodes') + ' -a'
-                cmd_out = self.du.run_cmd(self.hostname, cmd, sudo=True)
-                if cmd_out['rc'] == 0:
-                    for i in cmd_out['out']:
-                        if re.match(r'\s+Mom = ', i):
-                            mom_names = i.split(' = ')[1].split(',')
-                            for m in mom_names:
-                                if m in self.moms:
-                                    self.log_match(
-                                        "successfully sent hook file %s to %s"
-                                        % (hfile, m), interval=1)
-                                    self.log_match(
-                                        "successfully sent hook file %s to %s"
-                                        % (pyfile, m), interval=1)
-                                    self.moms[m].log_match(logmsg, starttime=t)
-                else:
-                    return False
+            if isinstance(attrs['event'], list):
```

Comment:
you can do like:
```python
_events = attrs['event']
if not isinstance(_events, (list,)):
    _events = [_events]
_events = [ x for x in _events if 'exec' in x ]
for _e in _events: # At this point _events will be list + it will have only mom hook event names
    '<code from "if 'exec' in hk:" condition>'
```

Reply:
Thanks for the suggestion. Will incorporate it